### ISSUE TITLE
What is the license of the OBOFoundry metadata?

### ISSUE THREAD

**Issue by andrawaag** [OBOFoundry/OBOFoundry.github.io#2274]
I am parsing https://obofoundry.org/registry/ontologies.jsonld to align with wikidata. It is not clear from the file, which license applies to the individual metadata file. It does list licenses for the individual ontologies. 

It has been suggested that the CC0 license applies to it, is this true? If so, may I ask to add that to the data, maybe similar to how Wikdiat does it?

![image](https://user-images.githubusercontent.com/770306/215104826-d4c8f539-bc4f-431b-ad43-b6176a29cb9c.png)

and
![image](https://user-images.githubusercontent.com/770306/215104899-1616bf96-ddc8-42c7-a6f4-da7cc3341010.png)

**Comment by jamesaoverton** [comment]
Yes, the OBO registry data is CC0 licensed. The LICENSE.md and LICENCE-code.md files at the top of this repo are intended to make this clear. This issue is also relevant https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1046.

Adding license info to the registry files themselves would be good. It would also be good to make this clear in the to-level README.md.
